conf/modules.config

```diff
@@ -53,4 +51,19 @@ process {
             ].join(" ")
         }
     }
+
+    withName: FASTQ_DL {
+        errorStrategy = 'ignore'
```

Comment:
Is `ignore` here because of the messy nature of download reads?

Reply:
Yes! But also that makes me think I should check what if the tool fails when there is no reads found/pulled or if it just exits.

If it just exits I can probably adjust it such that the fastq output is optional and then remove the error strategy